client/src/components/QA/QA.jsx

```diff
@@ -5,15 +5,70 @@ import QuestionList from './Questions/QuestionList.jsx';
 class QA extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
```

Comment:
looks good - though I think you'll probably delete this when you pull from the API

Reply:
Yes of course! its all for rendering right now